doc/guidance.md

```diff
@@ -14,9 +14,22 @@ This is about how to use Doxygen to maintain the API documents. There are three
 	- on mac, run "brew install graphviz"
 - goto the root directory of "amazon-freertos" (this will be referenced as $ROOT_AFR_DIR)
 	- cd $ROOT_AFR_DIR
-- run Doxygen command to generate the documents 
-	- doxygen doc/config/main
+- create a tag folder to store doxygen generated tags
+	- mkdir $ROOT_AFR_DIR/doc/tag
+- run Doxygen commands to generate the documents
```

Comment:
Does order matter? Nothing seems broken but doxygen complains it can't find some tag files. Although there are circular dependencies for a lot of these tags

Reply:
You have to run them in order to avoid the errors, but I beleive that there are circular links, which means you have to run them all twice to be sure all the links are good

Reply:
Done